src/arviz_plots/backend/__init__.py

```diff
@@ -99,6 +99,29 @@ def create_plotting_grid(
 
 
 # "geoms"
+def hist(x, height, target, *, width=1.6, bottom=None, align="center", color=None):
+    """Interface for a hist plot.
+
+    Parameters
+    ----------
+    x : float or array-like
+        The x coordinates of the bars.
+    height : float or array-like
+        The height(s) of the bars.
+    target : matplotlib.axes._axes.Axes
+        The target Axes object where the plot will be drawn.
+    width : float or array-like, default 0.8
+        The width(s) of the bars.
+    bottom : float or array-like, optional
+        The y coordinate(s) of the bottom side(s) of the bars.
+    align : {'center', 'edge'}, default 'center'
+        Alignment of the bars to the x coordinates.
+    color : color or array-like, optional
+        The colors of the bar faces.
```

Comment:
I might start with the same color for all rectangles, mostly because I am not sure how easy it will be to support the different color rectangles in all backends. If we then see it is feasible and of interest we can add that extra case.

There should also be a facecolor and an edgecolor arguments given there is a color argument and the generated form is "fillable" to match the expectations from https://arviz-plots.readthedocs.io/en/latest/api/backend/index.html#backend-interface-arguments